crates/bevy_ptr/README.md

```diff
@@ -6,8 +6,104 @@
 [![Docs](https://docs.rs/bevy_ptr/badge.svg)](https://docs.rs/bevy_ptr/latest/bevy_ptr/)
 [![Discord](https://img.shields.io/discord/691052431525675048.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/bevy)
 
-The `bevy_ptr` crate provides low-level abstractions for working with pointers in a more safe way than using rust's raw pointers.
+Pointers in computer programming, according to Wikipedia, are "objects in many programming languages that stores a memory address". 
```

Comment:
```suggestion
A pointer, according to Wikipedia, is an "object in many programming languages that stores a memory address". 
```

This fixes the plurality grammatical error where "stores" should be singular.

I would recommend rewriting this phrase, though. I don't think quoting Wikipedia is necessary here, especially with the "in many programming languages" part. It's implied that this is programming related, so the phrase is redundant.

Reply:
Agreed. Reworded this section.